src/lib/units/day-of-week.js

```diff
@@ -82,6 +82,19 @@ function parseWeekday(input, locale) {
     return null;
 }
 
+function parseIsoWeekday(input, locale) {
```

Comment:
I think falling back to parseWeekday is a bit excessive. If its string go through localeParse. If we now have number do the modulus. Return null if any of those fail.